src/frontend/src/utils/pipes.ts

```diff
@@ -84,3 +84,17 @@ export const numberParamPipe = (param: string | null) => {
     return null;
   }
 };
+
+/** Display timeline status in readable form
+ *  E.G. VERY_BEHIND -> Very Behind
+ */
+export const timelinePipe = (status: string) => {
+  var res = status.replaceAll('_', ' ');
+  res = res.toLowerCase();
+  res = res
+    .toLowerCase()
+    .split(' ')
+    .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
+    .join(' ');
+  return res;
```

Comment:
you could instead do: 
`res
.toLowerCase()
.split('_')
.map((s) => s.charAt(0).toUpperCase() +s.substring(1))
.join(' ')`